python/ccxt/base/exchange.py

```diff
@@ -2843,6 +2846,26 @@ def create_post_only_order(self, symbol, type, side, amount, price, params={}):
         query = self.extend(params, {'postOnly': True})
         return self.create_order(symbol, type, side, amount, price, query)
 
+    def create_stop_order(self, symbol, type, side, amount, price=None, stop_price=None, params={}):
```

Comment:
The `stop_price` argument must actually be `stopPrice` to make it consistent with the docs, otherwise we will have to write out different syntax for each language in the docs. We don't change the camelCaseNotation to underscore_snake_case_notation in in the argument names in general.